ui/src/app/pages/admin/admin-institution-edit.tsx

```diff
@@ -0,0 +1,411 @@
+            <TooltipTrigger data-test-id='tooltip' content={
+              errors && this.disableSave(errors) && <div>Answer required fields
+                <BulletAlignedUnorderedList>
+                  {errors.displayName && <li>Display Name should be of at most 80 Characters</li>}
+                </BulletAlignedUnorderedList>
+              </div>
+            } disable={isAddInstitution}>
+              <Button type='primary' disabled={this.disableSave(errors)} onClick={() => this.saveInstitution()}>
+                {this.buttonText}
+              </Button>
+            </TooltipTrigger>
+          </div>
+        </FlexRow>
+        <FlexRow>
+          <FlexColumn style={{width: '50%'}}>
+            <TextInputWithLabel
+                value={fp.startCase(institution.displayName)}
+                inputId='displayName'
+                inputName='displayName'
+                placeholder='New Username'
+                labelStyle={styles.label}
+                inputStyle={{width: '16rem', marginTop: '0.3rem'}}
+                labelText='Institution Name'
+                onChange={v => this.setState(fp.set(['institution', 'displayName'], v))}
+            />
+            <div style={{color: colors.danger}} data-test-id='displayNameError'>
+              {!isAddInstitution && errors && errors.displayName}
+              </div>
+            <label style={styles.label}>Institution Type</label>
+            <Dropdown style={{width: '16rem'}} data-test-id='role-dropdown'
+                      placeholder='Your Role'
+                      options={this.institutionTypeOptions}
+                      value={institution.organizationTypeEnum}
+                      onChange={v => this.updateInstitutionRole(v.value)}/>
+            <div style={{color: colors.danger}}>{!isAddInstitution && errors && errors.organizationTypeEnum}</div>
+
+            {showOtherInstitution && <TextInputWithLabel value={institution.organizationTypeOtherText}
+               onChange={v => this.setState(fp.set(['institution', 'organizationTypeOtherText'], v))}
+               inputStyle={{width: '16rem', marginTop: '0.8rem'}}/>}
+            <label style={styles.label}>Agreement Type</label>
+            <Dropdown style={{width: '16rem'}} data-test-id='agreement-dropdown'
+                      placeholder='Your Agreement'
+                      options={this.institutionAgreementTypeOptions}
+                      value={institution.duaTypeEnum}
+                      onChange={v => this.setState(fp.set(['institution', 'duaTypeEnum'], v.value))}/>
+            {institution.duaTypeEnum === DuaType.RESTRICTED && <FlexColumn data-test-id='emailAddress' style={{width: '16rem'}}>
+              <label style={styles.label}>Accepted Email Address</label>
+              <TextArea value={institution.emailAddresses && institution.emailAddresses.join(',\n')}
```

Comment:
the mocks do not have commas here

Reply:
Will ask Lou to update, he is aware that  i have added commas to make it easier for us to distinguish between two email addresses/domains